ArmoniK.Extension.CSharp.Client/ArmoniKClient.cs

```diff
@@ -38,54 +38,63 @@ namespace ArmoniK.Extension.CSharp.Client;
 /// </summary>
 public class ArmoniKClient
 {
-  private readonly ILogger                 _logger;
-  private readonly ILoggerFactory          _loggerFactory;
-  private readonly Properties              _properties;
-  private          IBlobService            _blobService;
-  private          ObjectPool<ChannelBase> _channelPool;
-  private          IEventsService          _eventsService;
-  private          IHealthCheckService     _healthCheckService;
-  private          IPartitionsService      _partitionsService;
-  private          ISessionService         _sessionService;
-  private          ITasksService           _tasksService;
-  private          IVersionsService        _versionsService;
+  private readonly ILogger        logger_;
+  private readonly ILoggerFactory loggerFactory_;
+  private readonly Properties     properties_;
```

Comment:
this should not happen with formatting...

Reply:
I PR will be made later to change .editorconfig